presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -231,6 +233,32 @@ public void testHiveViewInInformationSchema()
                 .contains(row("n_nationkey", "bigint", "", ""));
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testFailingHiveViewsForInformationSchema()
```

Comment:
Based on your comment, are you leaning toward not committing this test because you're worried about future versions of Coral supporting these functions? Perhaps you can instead use a construct that you think may never be supported by Coral? I like the idea of keeping the test if it's possible. FWIW, there are often times that tests need to be removed as functionality evolves.

Reply:
Added a more reliable test.